src/preferences.vala

```diff
@@ -20,26 +20,12 @@
 using Gee;
 
 public class Preferences : Object {
```

Comment:
I suggested to completely remove this file. Why do you think this is still needed?